MicroPython/scripts/upload_scripts.py

```diff
@@ -5,6 +5,7 @@
 import sys
 import git
 from git.refs.tag import TagReference
+from pathlib import Path
```

Comment:
Why?

Reply:
It will allow us to upload project from UNIX-like and Windows machine.